extension/src/config/config.js

```diff
@@ -16,17 +16,17 @@ function getCTPEnvCredentials () {
     projectKey: process.env.CTP_PROJECT_KEY,
     clientId: process.env.CTP_CLIENT_ID,
     clientSecret: process.env.CTP_CLIENT_SECRET,
-    apiUrl: 'https://api.sphere.io',
-    authUrl: 'https://auth.sphere.io'
+    apiUrl: process.env.CTP_HOST || 'https://api.sphere.io',
+    authUrl: process.env.CTP_AUTH_URL || 'https://auth.sphere.io'
   }
 }
 
 function getAdyenCredentials () {
   return {
     merchantAccount: process.env.ADYEN_MERCHANT_ACCOUNT,
     apiKey: process.env.ADYEN_API_KEY,
-    apiBaseUrl: process.env.ADYEN_API_BASE_URL || 'https://checkout-test.adyen.com/v40',
-    legacyApiBaseUrl: process.env.ADYEN_LEGACY_API_BASE_URL || 'https://pal-test.adyen.com/pal/servlet/Payment/v40'
+    apiBaseUrl: process.env.ADYEN_API_BASE_URL || 'https://checkout-test.adyen.com/v53',
```

Comment:
I was testing with v53 on the postman because of their open API spec, actually, I did not check it from the docs. Maybe we could ask Adyen for the difference ?

Reply:
Which spec are you talking about? Can you give a link?

Reply:
for now, the correct number seems to be v52. as all their documents and their code examples are using v52. will change this.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/e10699da3b521e2857864d1e0f992a1b3db9966a